boot/bootstate20.go

```diff
@@ -110,27 +116,143 @@ func (ks20 *bootState20Kernel) loadBootenv() error {
 		return fmt.Errorf("cannot use %s bootloader: does not support extracted run kernel images", bl.Name())
 	}
 
-	ks20.ebl = ebl
+	kmerki.ebl = ebl
```

Comment:
the code up to there needs to go back into the caller, otherwise how can it detect what kind of mutator to make

Reply:
sure, I was going to do that in the followup but it's fine to do this here instead.